tests/base/functions_12.cc

```diff
@@ -32,7 +32,7 @@ void check()
 
   std::vector<double> coeffs(n_mon);
   for (unsigned int i = 0; i < n_mon; ++i)
-    coeffs[i] = std::pow(-1.0,i)*(i+1);
+    coeffs[i] = std::pow(-1.0,static_cast<double>(i))*(i+1);
 
```

Comment:
I thought I fixed it by changing the first argument to a double a couple of days ago, but unsigned int for the second argument also didn't work...